src/index.html

```diff
@@ -5,8 +5,349 @@
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#" class="header__logo">
+        <img src="./images/logo.png" alt="Moyo logo">
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#apple" class="nav__link is-active">apple</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#samsung" class="nav__link">samsung</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#smartphones" class="nav__link">smartphones</a>
+          </li>
+
+          <li class="nav__item" data-qa="nav-hover">
+            <a href="#lp" class="nav__link">
+              laptops & computers
+            </a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#gadgets" class="nav__link">gadgets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#tablets" class="nav__link">tablets</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#photo" class="nav__link">photo</a>
+          </li>
+
+          <li class="nav__item">
+            <a href="#video" class="nav__link">video</a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="container">
+      <div class="card" data-qa="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac"
+          class="card__image"
+        >
+
+        <div class="card__bottom">
+          <h1 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h1>
+
+          <p class="card__id">Product code: 195434</p>
+
+          <div class="card__review">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+
+            <p class="card__review-count">Reviews: 5</p>
+          </div>
+
+          <div class="card__price">
+            <p class="card__price-title">Price:</p>
+
+            <p class="card__price-cost">$2,199</p>
+          </div>
+
+          <a
+            class="card__button"
+            type="bottom"
+            data-qa="card-hover"
+          >
+            Buy
+          </a>
+        </div>
+      </div>
+
+      <div class="card">
+        <img
+          src="./images/imac.jpeg"
+          alt="APPLE A1419 iMac"
+          class="card__image"
+        >
+
+        <div class="card__bottom">
+          <h1 class="card__title">
```

Comment:
Consider fixing the title further in the code

Reply:
fix it for all cards 
![image](https://github.com/mate-academy/layout_catalog/assets/97547824/733a05b8-b1f6-4124-b2f0-25a7198b045c)